common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -1104,7 +1104,7 @@ div.problem {
 
     .notification-message {
       display: inline-block;
-      width: flex-grid(6,10);
+      width: flex-grid(8,10);
```

Comment:
60% looks so odd to other users.  Why are optimizing the visual UX for a hidden button?